src/H5TSprivate.h

```diff
@@ -25,8 +25,9 @@
 #define H5TSprivate_H_
 
 #ifdef H5_HAVE_THREADSAFE
-/* Public headers needed by this file */
-#include "H5TSpublic.h" /* Public API prototypes */
+/* Include package's public headers */
+/* #include "H5TSpublic.h" */
```

Comment:
Same as in H5TSdevelop.h - any need to keep this line?

Reply:
Ditto - will take out